rpc/core/errors.go

```diff
@@ -17,7 +17,7 @@ var (
 	ErrTimedOutWaitingForTx    = errors.New("timed out waiting for tx to be included in a block")
 	ErrGenesisRespSize         = errors.New("genesis response is too large, please use the genesis_chunked API instead")
 	ErrChunkNotInitialized     = errors.New("genesis chunks are not initialized")
-	ErrNoChunks                = errors.New("no chunks")
+	ErrNoChunks                = errors.New("genesis file is small, therefore there are no chunks to serve. Please use the /genesis API instead")
```

Comment:
Can we make it so that if the genesis file is too small, we just return it when the first chunk is assessed? If so, we could deprecate `/genesis` completely. 

From the app developer's point of view, it's cumbersome to have to switch from one endpoint to another.

Reply:
It already behaves like that 😄 when the genesis file is small, querying the `/genesis_chunked` endpoint with `chunkID=0` will return the entire genesis file encoded in `base64`. See [here](https://github.com/cometbft/cometbft/blob/6927e7f967b919a0084edf08db9802aade59cb63/rpc/core/net.go#L136) .

The API returns `ErrNoChunks` when the genesis file is small and you query `genesis_chunked` with `chunkID >= 1`. The previous error msg was just `"no chunks"` which I thought wasn't very informative.